bin/live/pycbc_live_combine_single_fits

```diff
@@ -161,9 +161,11 @@ for ifo in args.ifos:
         cons_alpha = np.percentile(a, 100 - args.conservative_percentile)
         cons_alphas_out[ifo][counter] = cons_alpha
         alphas_out[ifo][counter] = mean_alpha
-        cons_count = np.percentile(c, args.conservative_percentile)
-        cons_counts_out[ifo][counter] = cons_count * len(c)
-        counts_out[ifo][counter] = c.sum()
+        # To get the count values, we need to convert to rates and back again
+        r = c / l_times[ad_order]
+        cons_rate = np.percentile(r, args.conservative_percentile)
+        cons_counts_out[ifo][counter] = cons_rate * l_times[ad_order].sum()
+        counts_out[ifo][counter] = np.average(r) * l_times[ad_order].sum()
```

Comment:
Is there a specific reason to use `np.average(r)` and not `np.mean(r)`, as used elsewhere in the code?

Reply:
I was experimenting with using `average(c, weights=l_times[ad_order])`, but then realised that it wasnt right, we can switch

```suggestion
        counts_out[ifo][counter] = np.mean(r) * l_times[ad_order].sum()
```